microshift.sh

```diff
@@ -103,6 +103,7 @@ EOF
     sed -i "/--bootproto=dhcp/a\network --hostname=api.${SNC_PRODUCT_NAME}.${BASE_DOMAIN}" scripts/image-builder/config/kickstart.ks.template
     sed -i 's/clearpart --all --initlabel/clearpart --all --disklabel gpt/g' scripts/image-builder/config/kickstart.ks.template
     sed -i "/clearpart --all/a\part biosboot --fstype=biosboot --size=1" scripts/image-builder/config/kickstart.ks.template
+    sed -i '$e grub2-install --target=i386-pc /dev/vda' scripts/image-builder/config/kickstart.ks.template
```

Comment:
Have you tried using https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/performing_an_advanced_rhel_9_installation/kickstart-commands-and-options-reference_installing-rhel-as-an-experienced-user#bootloader-required_kickstart-commands-for-handling-storage ?

Reply:
Praveen tried adding `bootloader` to the kickstart file, but for some reason this was not equivalent to manuallly running `grub2-install`.